server/users/schemas/user.schema.ts

```diff
@@ -16,6 +16,9 @@ export class User {
     @Prop({ required: true, unique: true })
     oryId: string;
 
+    @Prop({ required: true })
+    role: string;
```

Comment:
We shouldn't store the roles in our DB. Instead, use ory and collect that information form the session.

One of the reasons for that is because this information need to be consistent and being able to decode that from the session means that we also can do it from the frontend clients.